nav2_collision_monitor/include/nav2_collision_monitor/polygon.hpp

```diff
@@ -243,6 +254,9 @@ class Polygon
   /// @brief Polygon publisher for visualization purposes
   rclcpp_lifecycle::LifecyclePublisher<geometry_msgs::msg::PolygonStamped>::SharedPtr polygon_pub_;
 
+  /// @brief Polygon velocity (if any)
+  std::vector<std::shared_ptr<PolygonVelocity>> polygon_velocity_;
```

Comment:
Since it is vector of `PolygonVelocity` type, this, I think, is better to rename to something, like:
```
std::vector<std::shared_ptr<PolygonVelocity>> polygons_velocity_;
```
... in order to avoid further confusions